chain/network/src/peer/codec.rs

```diff
@@ -15,19 +15,14 @@ use near_rust_allocator_proxy::allocator::get_tid;
 use crate::stats::metrics;
 use crate::types::PeerMessage;
 
```

Comment:
can you also add a small 'file-level' comment on what this codec does?

From what I see, it is responsible for reading & writing the 'network' messages from the stream (encoded as 
- 4 bytes: length of the message
-  message itself

And maybe also add the comment to is_forward_tx -> that AFAIK is trying to figure out if transaction is forwarded without borsch-deserializing the whole context.

(btw - we should have a unittest in this file that makes sure that this function keeps working - as it looks very brittle and easily breakable)

Reply:
There are units tests for `Codec` in this file. Later, we should review if any cases are missing.

Reply:
@mm-near Thanks, I added documentation.

Reply:
I added documentation about `is_forward_tx` to https://github.com/near/nearcore/pull/5409/files